moneta-convert/moneta-convert-ecb/src/main/java/org/javamoney/moneta/convert/ecb/ECBAbstractRateProvider.java

```diff
@@ -120,7 +120,7 @@ public ExchangeRate getExchangeRate(ConversionQuery conversionQuery) {
                 }
                 RateResult result = findExchangeRate(conversionQuery);
 
-                ExchangeRateBuilder builder = getBuilder(conversionQuery, result.date);
+                ExchangeRateBuilder builder = getBuilder(conversionQuery);
```

Comment:
I'm not sure about this code but it might be that result date is not the same as requested date. For example we requested the rate for now but it returns a rate which started on today's midnight.
Could you please check the logic?

Reply:
ok, but again this may be a smell of some bug (I hope it's not)